src/main/java/uk/ac/sanger/sccp/stan/service/PermServiceImp.java

```diff
@@ -182,6 +233,7 @@ public void validateAddresses(Collection<String> problems, Labware lw, Collectio
     public void validatePermValues(Collection<String> problems, Collection<PermData> permData) {
```

Comment:
Comment not updated for Valid - Control barcode  with a positive control type

Reply:
Fair enough. I have revised the comment.